interview.md

```diff
@@ -11,41 +11,61 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+A DSL built to help poker players keep track of data at the poker table in real time. This dsl would help reinvent poker analyis for poker players. Right now player rely on their memory to write down what happened on any given hand durring game. They may wait till break to add a particular interesting hand into thier notes, or keep a mental note of how a player plays durring a hand, or even wait many hours until the event has stopped or they got busted out to write down things that happened at the table all off of memory. Instead image a DSL for poker players that allows them to keep track of poker related events at the poker table, and even provide them with statistics so they can make their best move.
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+This DSL is aproprite for poker player so that they can input events that happen at the table in real time and allows for more accurate data represention then when they add it to thier notes after the fact. Additionally I keeps all of a players notes, poker data, and suggestions all in one place. The need is for real time note / obervation taking while sitting at the poker table. It would allow the players to take accurate observation in real time but also not take away from their attention to the table. 
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+This idea exites me and other poker players because it combines this notion of a ststitical game with more accurate stastitics as well as provides the player the ability to keep track of past hands without having to recall memory from the last 4 hours of play. I was able to come up with this idea after interveiwing my dad who is a sofware engineer and a poker player. After having a long converstion about things he does at work and things that he does in his free time we narrowed in on poker. Since we both play poker we both found it difficult to keep track of things that happen at the poker table espessally since you can be so focused on the game. Additionally good pker player always write down intersting events that happen at the table such as big hands, how they busted out, and tells of oher players. This idea of a dsl that allows for real time tracking of these events came to life. 
+
```

Comment:
This is very cool that you and your Dad came up with it. The time scale is also an important component of this project - that it isn't a hand or two, but hours worth of information that the user would want to track.